daemon/daemon.go

```diff
@@ -47,7 +48,10 @@ type Daemon struct {
 	tomb         tomb.Tomb
 	router       *mux.Router
 	capRepo      *caps.Repository
-	asserts      *asserts.Database
+	skills       *skills.Repository
+	// enableInternalSkillActions controls if adding and removing skills and slots is allowed.
```

Comment:
a  very silly thing: can you move this flag and the comment to the end of the struct? take `skills` with it if you want to keep them close :-)